cms/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM hmccaa/hmcc-web:strapi-0.1.1
+FROM hmccaa/hmcc-web:strapi-0.1.0-arm
```

Comment:
Revert this change. we do want want this to change because different people use different os/cpu platform